### DISCUSSION TITLE
How to display current function document if not enable lsp-ui (because performance issue)?

### DISCUSSION THREAD

**Discussion by zw963** [emacs-lsp/lsp-mode#3399] [Q&A] [answered]
As eglot--hover-info does or when use with lsp-ui, hover on a function, we can see the niced look function definition.

Thank you.

**Comment by zw963** [comment, marked as answer]
For make this question more clearly, what i want is, a lisp function, which i can bind to a key or run it directly from M-x to display document on i want it to be. (still the performance reason)

**Reply by yyoncho** [reply]
M-x lsp-describe-thing-at-point
